.github/workflows/gh-ci.yaml

```diff
@@ -22,7 +22,7 @@ defaults:
 
 env:
   MDA_CONDA_MIN_DEPS: "pip pytest mmtf-python biopython networkx cython matplotlib-base scipy griddataformats hypothesis gsd codecov threadpoolctl"
-  MDA_CONDA_EXTRA_DEPS: "seaborn>=0.7.0 clustalw=2.1 netcdf4 scikit-learn joblib>=0.12 chemfiles>=0.9 tqdm>=4.43.0 tidynamics>=1.0.0 rdkit>=2020.03.1 h5py openmm"
+  MDA_CONDA_EXTRA_DEPS: "seaborn>=0.7.0 clustalw=2.1 netcdf4 scikit-learn joblib>=0.12 chemfiles>=0.9,<=0.10.0 chemfiles-lib>=0.9,<=0.10.0 tqdm>=4.43.0 tidynamics>=1.0.0 rdkit>=2020.03.1 h5py openmm"
```

Comment:
You also want to pin `chemfiles-python` here

Reply:
Or even simpler, you can replace `chemfiles` by `chemfiles-python`.

Reply:
That works, thanks!

Reply:
Another alternative I just though about would be to use the pip version of chemfiles (which comes with its own pre-built chemfiles-lib instead of relying on a separate package), and would allow to install 0.10.1 which contains quite a few bug fixes, in particular for mmCIF which might be of interest for MDA.

Reply:
Thanks I'll add trying the switch to pip to the to-do list.